libs/shared/data-access/work-api/src/lib/services/work-api-routes.service.ts

```diff
@@ -360,4 +361,11 @@ export class WorkApiRoutesService extends HttpBaseService {
       map(dashboardsSchema => dashboardsSchema.map(dashboardSchema => new DashboardModel(dashboardSchema))),
     );
   }
+
+  /**
+   * Gets counts for each dashboard tab
+   */
+  public getDashboardCounts$(transactionSetKey: string): Observable<DashboardCountModel[]> {
+    return this._handleGet$<DashboardCountModel[]>(`/v1/admin/dashboards/${transactionSetKey}/counts`);
```

Comment:
You should convert to a model here
```suggestion
    return this._handleGet$<DashboardCountModel[]>(`/v1/admin/dashboards/${transactionSetKey}/counts`).pip(map => new Model);
```